node/src/p2p.rs

```diff
@@ -240,13 +241,22 @@ impl P2p {
         let peer_tracker = Arc::new(PeerTracker::new(args.event_pub.clone()));
         let peer_tracker_info_watcher = peer_tracker.info_watcher();
 
-        let mut worker = Worker::new(args, cmd_rx, header_sub_tx, peer_tracker)?;
+        let cancellation_token = CancellationToken::new();
```

Comment:
did you intend to include changes wrt shutdown on this pr?

Reply:
no. removed